src/platform/orchestrator/src/MpiServer.cpp

```diff
@@ -60,130 +63,69 @@ class PlatformLog
     static OSCONFIG_LOG_HANDLE m_log;
 };
 
-OSCONFIG_LOG_HANDLE PlatformLog::m_log = nullptr;
+OSCONFIG_LOG_HANDLE PlatformLog::m_log = NULL;
 
-enum class StatusCode
+enum StatusCode
 {
     OK = 200,
     BAD_REQUEST = 400,
     NOT_FOUND = 404,
     INTERNAL_SERVER_ERROR = 500
 };
 
-class Server
+const char* CreateUuid()
```

Comment:
Since function allocates mem for return, the return value cannot be const.

Also, function as it is invoked only from this file, can be static.